index.js

```diff
@@ -84,28 +88,28 @@ exports.unique = function unique(arr) {
   return resultArr;
 };
 
-exports.common = function common(arr) {
-  assert(typeof arr === 'object' && arr.hasOwnProperty('length'), 'array required');
```

Comment:
Maybe it shouldn't be array, it should pass the args one by one, just like `.inpath`.